src/torchmetrics/metric.py

```diff
@@ -779,148 +783,151 @@ def __hash__(self) -> int:
         return hash(tuple(hash_vals))
 
     def __add__(self, other: "Metric") -> "Metric":
-        """Construct conpositional metric using the addition operator."""
+        """Construct compositional metric using the addition operator."""
         return CompositionalMetric(torch.add, self, other)
 
     def __and__(self, other: "Metric") -> "Metric":
-        """Construct conpositional metric using the logical and operator."""
+        """Construct compositional metric using the logical and operator."""
         return CompositionalMetric(torch.bitwise_and, self, other)
 
-    def __eq__(self, other: "Metric") -> "Metric":
-        """Construct conpositional metric using the equal operator."""
+    # Fixme: this shall return bool instead of Metric
```

Comment:
this likely isn't relevant any more since it's now established behavior
```suggestion
```

Reply:
Established = wanted/desired?